src/network/networkd-link.c

```diff
@@ -1442,8 +1456,7 @@ static int link_acquire_ipv6_conf(Link *link) {
                         return log_link_warning_errno(link, r,  "Could not acquire DHCPv6 lease: %m");
         }
 
-        if (link_ipv6_accept_ra_enabled(link)) {
-                assert(link->ndisc_router_discovery);
```

Comment:
At `link_configure()`  if `link_is_bridge_port` we doing kind of sanity already. if same condition both places.
Will add back .